draft-ietf-quic-http.md

```diff
@@ -455,13 +515,13 @@ superseded by QUIC transport parameters in HTTP-over-QUIC. Below is a listing of
 how each HTTP/2 SETTINGS parameter is mapped: 
 
   SETTINGS_HEADER_TABLE_SIZE:
-  : Sent in SETTINGS frame.
+  : An integer with a maximum value of 2^32 - 1.
```

Comment:
now that these are variable length does it need a max size? We can merge extended settings and talk about that as a different issue too